docs/design-survey-predefine-answers.md

```diff
@@ -1,23 +1,99 @@
+- [Multiple Question Values](#multiple-question-values)
+- [Default Question Values](#default-question-values)
+
+## Single Question Value
+
+You can set a question's [`value`](https://surveyjs.io/form-library/documentation/questiontextmodel#value) property directly to populate a single form field. Call the Survey's [`getQuestionByName(questionName)`](https://surveyjs.io/form-library/documentation/surveymodel#getQuestionByName) method to obtain the question's instance and then set the `value` property on this instance:
 
 ```js
+import { Model } from "survey-core";
+
 const surveyJson = {
   "elements": [{
     "name": "subscribed",
-    "type": "checkbox",
+    "type": "boolean",
+    "renderAs": "checkbox",
     "title": "I agree to receive weekly newsletters"
-    "defaultValue": true
   }]
 }
+
+const survey = new Model(surveyJson);
+const subscribedQuestion = survey.getQuestionByName("subscribed");
+subscribedQuestion.value = true;
 ```
 
-You can set the question `value` property or call the Survey's [`setValue(questionName, newValue)`](https://surveyjs.io/Documentation/Library?id=surveymodel#setValue) method to change the value at any point in your application:
+Alternatively, you can call the Survey's [`setValue(questionName, newValue)`]() method:
 
 ```js
 import { Model } from "survey-core";
 
+const surveyJson = {
+  // ...
+}
+
 const survey = new Model(surveyJson);
 survey.setValue("subscribed", false);
-```
\ No newline at end of file
+```
+
+## Multiple Question Values
+
+To populate multiple form fields, use the [`data`](https://surveyjs.io/Documentation/Library?id=surveymodel#data) property of a Survey instance. This property contains survey results data as an object in which keys are question names and values are answers. You can assign a new object to `data` property:
+
+```js
+import { Model } from "survey-core";
+
+const surveyJson = {
+  "elements": [{
+    "name": "firstName",
+    "title": "First Name"
+  }, {
+    "name": "lastName",
+    "title": "Last Name"
+  }]
+}
+
+const survey = new Model(surveyJson);
+survey.data = {
+  "firstName": "John",
+  "lastName": "Doe"
+}
+```
+
+The code above *replaces* the old `data` object and erases entered data (if there was any). If you want to *merge* the new and old objects, call the [`mergeData(newDataObj)`](https://surveyjs.io/form-library/documentation/surveymodel#mergeData) method:
+
+```js
+import { Model } from "survey-core";
+
+const surveyJson = {
+  // ...
+}
+
+const survey = new Model(surveyJson);
+survey.mergeData({
+  "lastName": "Doe"
+});
+```
+
+## Default Question Values
+
+You can set a question's [`defaultValue`](https://surveyjs.io/Documentation/Library?id=Question#defaultValue). It will be used until a proper [`value`](https://surveyjs.io/Documentation/Library?id=Question#value) is specified by a user or programmatically. If the proper `value` is never specified, `defaultValue` is saved in the survey results.
```

Comment:
It's not a good idea to use API member names as the corresponding phrase in a sentence. If you'd like to use the property name as is, it's better to say `PropertyName` property. For example:

> You can set a question's `defaultValue` property.

Otherwise, it's better to use a fully-functional phrase as a hyperlink anchor:

```suggestion
You can set a question's [`default value`](https://surveyjs.io/Documentation/Library?id=Question#defaultValue). It will be used until a proper [`value`](https://surveyjs.io/Documentation/Library?id=Question#value) is specified by a user or programmatically. If the proper `value` is never specified, `defaultValue` is saved in the survey results.
```

Reply:
I can't agree with you here. When you read, you spell out property names as if they are regular phrases, so treating them as such is only natural. Splitting property names by words or otherwise modifying them is not an option because people use them in code verbatim.